### PR TITLE
Consider flake8 rules

### PR DESCRIPTION
**Pull request by amureki**
This PR:
1. Adds flake8 pre-commit hook
2. Fixes flake8 issues in main package files
3. Fixes flake8 issues in test files

### CODE REVIEW

**Comment by amureki** [comment]
@anapaulagomes sorry for the big one right here. :(

There are some small code changes to follow flake8 apart from just styling fixes.

File: model_mommy/mommy.py [hunk 1]
```diff
@@ -417,7 +454,7 @@ def get_required_values(generator, field):
     If required value is a string, simply fetch the value from the field
     and return.
     '''
-    #FIXME: avoid abreviations
+    # ¬FIXME: avoid abreviations
```
Comment (anapaulagomes) [inline model_mommy/mommy.py]: I might be missing something but why do we need this symbol? 😅

File: test/generic/tests/test_mommy.py [hunk 2]
```diff
@@ -296,8 +298,10 @@ def test_nullable_many_to_many_is_not_created_even_if_flagged(self):
     def test_m2m_changed_signal_is_fired(self):
         # Use object attrs instead of mocks for Django 1.4 compat
```
Comment (anapaulagomes) [inline test/generic/tests/test_mommy.py]: Shall we add a `TODO` here too?

File: model_mommy/mommy.py (hunk 1 above)
Comment (amureki) [reply model_mommy/mommy.py]: @anapaulagomes I wonder what is up with my personal laptop lately with all that symbols appearing here and there, really 😳

File: test/generic/tests/test_mommy.py (hunk 2 above)
Comment (amureki) [reply test/generic/tests/test_mommy.py]: Yeah, this looks like a TODO, right 👍

**Pull request merged**